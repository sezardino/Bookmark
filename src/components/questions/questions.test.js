import React from 'react';
import {shallow} from 'enzyme';

import Questions from './questions';

describe('Component Questions', () => {
	const mockData = [
		{question: 'question', answer: 'answer', id: '1'},
		{question: 'question', answer: 'answer', id: '1'},
		{question: 'question', answer: 'answer', id: '1'},
	];
	const component = shallow(<Questions data={mockData} />);

	it('Component Questions rendered correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
