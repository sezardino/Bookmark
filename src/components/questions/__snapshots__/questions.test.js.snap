// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Questions Component Questions rendered correctly 1`] = `
"<Fragment>
  <ul className=\\"questions faq__questions\\" />
  <button type=\\"button\\" className=\\"button button--blue faq__button\\" onClick={[Function: onClick]}>
    More Questions
  </button>
</Fragment>"
`;
